.github/workflows/build-push-images-pr.yaml

```diff
@@ -34,6 +134,13 @@ jobs:
           context: .
           file: deployment/docker/Dockerfile
           push: true
+          target: prod
           tags: ${{ steps.docker_meta.outputs.tags }}
+          labels: ${{ steps.docker_meta.outputs.labels }}
+          cache-from: |
+            type=gha,scope=test
+            type=gha,scope=prod
+          cache-to: type=gha,scope=prod
```

Comment:
We only push the `prod` target image.

This step will reuse the previous cache in the previous job. We already build the image, so we only push this time.